odo/backends/tests/test_sql.py

```diff
@@ -305,24 +305,15 @@ def test_discover_oracle_intervals(freq):
         (sa.types.NullType, string),
         (sa.REAL, float32),
         (sa.Float, float64),
-        pytest.param(
-            sa.Float(precision=24), float32,
-            marks=pytest.mark.xfail(
-                reason=dedent("""
-                Currently returns float64.
-                Unique instances of sa.Float(precision=24) do not equate to each other, which prevents
-                discover_typeengine from correctly keying into sql.revparses.
-                """).strip()
-            )
-        ),
+        (sa.Float(precision=24), float32),
```

Comment:
should we also test precision=53 and something that is not one of the hard coded numbers?

Reply:
The `precision=53` case is defined in the next line.
Good idea about adding a test for another number.

Reply:
The expected case for `Float(precision=42)` would depend on the answer to the questions about behavior here, https://github.com/blaze/odo/pull/592#discussion_r145212373